hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/WritableAccountStore.java

```diff
@@ -43,6 +45,7 @@
  * class is not complete, it will be extended with other methods like remove, update etc.,
  */
 public class WritableAccountStore extends ReadableAccountStoreImpl {
+    private static final Logger log = LogManager.getLogger(WritableAccountStore.class);
```

Comment:
Leftover?

Reply:
Meh doesn't matter I guess, hard to believe it has any measurable overhead. 😁

Reply:
Removed.